spec/features/user_can_sign_up_spec.rb

```diff
@@ -0,0 +1,46 @@
+feature 'user can sign up' do
+  let(:user) {create(:user, email: "gittan@gmail.com")}
+    before do
+    visit root_path
+  end
+  
+  
+  context 'successfully created a new account' do
+    before do
+      click_on "Sign up"
+    end
+
+    it 'User should see success message' do
+      fill_in "Name", with: "fredagsmys"
+      fill_in "Email", with: "myspys@gmail.com"
+      fill_in "Password", with: user.password
+      fill_in "Password confirmation", with: user.password_confirmation
+      click_on "SignUp"
+      expect(page).to have_content "You have signed up successfully"
+    end
+  end
+
+  context 'couldnt created a new account' do
+    before do
+      click_on "Sign up"
+    end
+
+    it 'User should see error message' do
```

Comment:
change this to a more descriptive title like "User sees error message when the password doesn't match password confirmation"